boltzrpc/client/connection.go

```diff
@@ -26,11 +28,16 @@ type Connection struct {
 }
 
 func (connection *Connection) Connect() error {
-	creds, err := credentials.NewClientTLSFromFile(connection.TlsCertPath, "")
+	creds := insecure.NewCredentials()
+	var err error
```

Comment:
Why is `err` declared here?

Reply:
So that we can do a `=` assignment to the `creds` var from line 31.